### DISCUSSION TITLE
spatie/medialibrary package support?

### DISCUSSION THREAD

**Discussion by zhanang19** [filamentphp/filament#283] [Ideas]
Great work for @danharrin and other collaborators!

Just curious to know, is there any plan to add support for [spatie/medialibrary](https://github.com/spatie/laravel-medialibrary) package to filament?

**Comment by danharrin** [comment]
Yes there is!

**Reply by yob-yob** [reply]
perfect... 

support for a field that selects pre-uploaded files would also be a great match.

**Reply by zhanang19** [reply]
Great, thanks for let me know!
